sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiverClient.cs

```diff
@@ -493,45 +493,45 @@ public virtual async Task<ServiceBusMessage> PeekAsync(CancellationToken cancell
         /// Completes a <see cref="ServiceBusMessage"/> using its lock token. This will delete the message from the service.
         /// </summary>
         ///
-        /// <param name="lockToken">The lock token of the corresponding message to complete.</param>
+        /// <param name="message">The lock token of the corresponding message to complete.</param>
         /// <param name="cancellationToken"></param>
         ///
         /// <remarks>
-        /// A lock token can be found in <see cref="ServiceBusMessage.SystemPropertiesCollection.LockToken"/>,
+        /// A lock token can be found in <see cref="ServiceBusReceivedMessage.LockToken"/>,
```

Comment:
nit: we need to update the docs as well.

Reply:
Updated